content/blog/2019-08-alphas.md

```diff
@@ -0,0 +1,231 @@
+
+# The road so far
+
+Tokio was first announced almost exactly [three years
+ago][tokio-ann]. When Tokio was first released, it contained a minimal, single
+threaded, scheduler and APIs for TCP. Since then, it has grown to include a
+multi-threaded work-stealing scheduler, APIs for timers, TCP, UDP, UDS, file
+system access, signal handling, managing child processes, high performance async
+aware channels, and more. All this thanks to a [dedicated team of
+maintainers][maint] and an [active community of contributors][contrib]. Tokio
+would not be where it is without you! Thank you so much.
+
+The goal was (and still is) to provide an ergonomic library for writing
+asynchronous applications for Rust without compromising on performance. The past
+three years has been spent building out the Rust I/O ecosystem and discovering
+idioms and patterns for writing idiomatic asynchronous Rust. Over the past three
+years, the Rust community built a vibrant ecosystem around Tokio.
+
+Some libraries include:
+
+- an [HTTP client and server][hyper] (with [HTTP/2.0 support][h2]).
+- web frameworks ([warp][warp], [actix][actix]).
+- a [QUIC][quinn] implementation.
+- [composable components][tower] for building robust servers and clients.
+- a [gRPC client and server][grpc]
+
+[hyper]: https://github.com/hyperium/
+[h2]: https://github.com/hyperium/h2
+[quinn]: https://github.com/djc/quinn
+[tower]: https://github.com/tower-rs/tower
+[warp]: https://github.com/seanmonstar/warp/
+[actix]: https://github.com/actix/actix-web
+[grpc]: https://github.com/tower-rs/tower-grpc/
+
+(and of course, there are many more).
+
+The number of users has been growing too! Users include:
+
+- [Linkerd](https://linkerd.io/)
+- [Microsoft](https://github.com/Azure/iotedge)
+- [Amazon](https://github.com/firecracker-microvm/firecracker)
+- [Facebook](https://github.com/facebookexperimental/mononoke)
+- [Vector](https://vector.dev/)
+- [Deno](https://github.com/denoland/deno)
+- [Libra](https://github.com/libra/libra)
+- [PingCAP](https://pingcap.com)
+- [Parity](https://www.parity.io/)
+
+Over the past three years, the Rust team has not been idle either. They have
+been working tirelessly on a huge feature that will greatly improve the
+experience of writing asynchronous code with Rust: [async & await
+syntax][async_await]. This new syntax is about to land on Rust stable.
+
+All of Tokio's iteration to date was done while maintaining API stability. Tokio
+has not had a breaking release since the very first 0.1.0 release on September
+9, 2016. Understandibly, a long list of changes to make has been building up.
+The introduction of async & await syntax is the right moment to make these
+breaking changes.
+
+# What we have now
+
+Today's release of Tokio 0.2.0-alpha.1 with async & await updates all of Tokio's
+components to use [`std::future`][std_future] and all usage of `futures` 0.1 has
+been removed.
+
+Changes include:
+
+**Using async fns for asynchronous operations.**
```

Comment:
It would be super neat to have an example of `#[tracing::instrument]` on an `async fn` as well (which should be on crates.io by the end of the day)...